NSS-Spontaneous-Spoonbills/Sprint3
# Edit account information

## Link to Ticket

Closes #14 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout cr_editAccount
python manage.py runserver
```
1. Login
1. Navigate to my account by clicking nav bar.
1. Click on edit account button
1. Should have a form to edit multiple fields
1. Click update button when done
1. Should be taken back to account home with the new updated information showing
1. Logout and log back in, the new adjustments should still be there
1. Logout and try and navigate to accountedit/1/ should be redirected to login
1. After login you should start at the edit form
1. Ship my beautiful work into master with a <3


## Mentions @username

@MrErin @obscure-shadow @JoeShep 

## Testing


## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
